packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -1520,7 +1547,7 @@ export class SceneLoader {
      * @param onError a callback with the scene, a message, and possibly an exception when import fails
      * @param pluginExtension the extension used to determine the plugin
      * @param name defines the name of the file, if the data is binary
-     * @returns The loaded plugin
+     * @deprecated Please use ImportMeshAsync instead

```

Comment:
Maybe we should mark the whole class as deprecated and point people to the module level functions, and remove @experimental from them now?

Reply:
I do not want to break the entire classes.